homeassistant/components/homematic/cover.py

```diff
@@ -48,6 +48,7 @@ def is_closed(self):
         """Return if the cover is closed."""
         if self.current_cover_position is not None:
             return self.current_cover_position == 0
+        return None
```

Comment:
Revert this file -- this change isn't necessary.

Reply:
It doesn't change the functionality, but it didn't comply with [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) without it